modules/rosa-checking-account-version-info-cli.adoc

```diff
@@ -3,7 +3,7 @@
 // * rosa_cli/rosa-checking-acct-version-cli.adoc
 
 [id="rosa-checking-account-version-information_{context}"]
-= Checking account and version information with the rosa CLI
+= Checking account and version information with the {product-title} (ROSA) CLI (`rosa`)
```

Comment:
This will make the title very long. Let's use just "CLI" in the title and alter the first sentence of the module to refer to the expanded version.